checkov/policies3d/runner.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import annotations
+
+import logging
+from enum import Enum
+
+from checkov.common.bridgecrew.check_type import CheckType
+from checkov.common.models.enums import CheckResult
+from checkov.common.output.record import Record
+from checkov.common.output.report import Report
+
+from checkov.common.runners.base_post_runner import BasePostRunner
+from checkov.common.util.type_forcers import force_list
+from checkov.policies3d.checks_infra.base_check import Base3dPolicyCheck
+from checkov.runner_filter import RunnerFilter
+
+
+class CVEAttribute(str, Enum):
+    RISK_FACTORS = "risk_factors"
+
+
+class Policy3dRunner(BasePostRunner):
+    check_type = CheckType.POLICY_3D  # noqa: CCE003  # a static attribute
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def run(  # type:ignore[override]
+            self,
+            checks: list[Base3dPolicyCheck] | None = None,
+            scan_reports: list[Report] | None = None,
+            runner_filter: RunnerFilter | None = None
+    ) -> Report | list[Report]:
```

Comment:
It looks like that only a scan report is returned, in which case it's a list of reports?

Reply:
I extended to list in case its relevant in the future but you're right that it's not needed. In second thought it can be removed now and if it'd be relevant in the future it'd be added, so removing